lib/features/song_detail/presentations/song_details_data.dart

```diff
@@ -0,0 +1,24 @@
+class DummyData {
+  final String titles = 'KJ NO.999';
+
+  final String subtitles = 'KAMI PUJI DENGAN RIANG DIKAY ALLAH YANG BESAR';
+
+  List<String> verse = [
+    'verse 1',
+    'verse 2',
+    'back to verse 1',
+    'back to verse 3',
+    'chorus'
+  ];
+
+  List<String> data = [
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ',
+    'at. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequ'
+  ];
+
```

Comment:
nah skarang gabung yang variable verse deng data, bekinng model class baru jo d p nama SongPart d p attribute pake String type, String content.
Jadi yang di class ini d p tipe data for variabel itu jadi List<SongPart>

nanti di screen torang kurang mo looping dari variabel list ini for mo tampilkan dia verse/chorus/ dll